templates/iucr/manuscript.tex

```diff
@@ -77,9 +77,7 @@ \section{Section title}
 
 \subsection{Subsection title}
 
-Text text text text text text text text text text text text text text
-text text text text text text text.
-
+Citing is a practice to acknowledge and document the sources of information or evidence, e.g. \cite{billinge_problem_2007} and \cite{keen_total_2020} , that you use to support your arguments or statements in an article, ensuring proper credit is given and helping readers verify the original materials.
```

Comment:
we don't need this here.  People don't need instructions how to write their paper.